pages/common/mixxx.md

```diff
@@ -0,0 +1,28 @@
+# mixxx
+
+> Free and open source cross-platform DJ software <https://www.mixxx.org>.
+> Command-line arguments are documented at <https://mixxx.org/manual/latest/chapters/appendix.html#command-line-options>.
+
+- Start the Mixxx GUI in full-screen:
+
+`mixxx --fullScreen`
+
+- Start in safe developer mode to debug a crash:
+
+`mixxx --developer --safeMode`
+
+- Debug a malfunction:
+
+`mixxx --debugAssertBreak --developer --loglevel trace`
+
+- Start mixxx using a contained configuration:
+
+`mixxx --resourcePath {{mixxx/res/controllers}} --settingsPath {{settingsPath}}`
+
+- Debug a custom controller mapping:
+
+`mixxx --controllerDebug --resourcePath {{custom/mapping/directory}}`
```

Comment:
```suggestion
`mixxx --controllerDebug --resourcePath {{path/to/directory}}`
```

Reply:
I thought the example argument may also convey some information. Mappers may not always be geeks ;)

Did I miss something about specifying example args/paths in the guidelines, btw?

Reply:
Generally, we try and ensure pages are consistent with each other. In this case, in other pages we usually use `path/to/file` or `path/to/directory` or even `path/to/descriptive_word.some_extension`.

In this case, it sounds like a mapping directory is a domain-specific term. Maybe it's ok in this instance to leave this as-is.